src/main/java/emissary/admin/Startup.java

```diff
@@ -413,6 +418,24 @@ protected void stopAndWaitForPlaceCreation() {
             numPlacesFound = this.places.size();
 
             if (numPlacesFound >= numPlacesExpected) {
+
+                if (this.node.isStartUpMode() &&
+                        this.failedPlaces.size() >= 1) {
+
+                    StringBuilder failedPlaceList = new StringBuilder();
+                    failedPlaceList.append(
+                            "Server failed to start due to Strict mode being enabled.  To disable strict mode, " +
+                                    "run server start command without the --strict flag.\n");
+                    failedPlaceList.append("The following places have failed to start: \n");
```

Comment:
I think from a log aggregation perspective I would rather not have new line's in the log message. 
Unless it makes it totally un-readable.

Reply:
I agree, a "; " should suffice.